modules/apps/object/object-rest-test/src/testFunctional/tests/ScopeObjectAPIsForDifferentInstances.testcase

```diff
@@ -1,20 +1,22 @@
 @component-name = "portal-headless"
 definition {
 
-	property custom.properties = "include-and-override=portal-liferay-online.properties";
+	property custom.properties = "include-and-override=portal-liferay-online.properties${line.separator}virtual.hosts.valid.hosts=localhost,127.0.0.1,www.able.com";
 	property portal.release = "true";
```

Comment:
The others I think it's good, but I found the custom.properties you also set in every test cases, is it necessary?